src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -0,0 +1,307 @@
+import React, {forwardRef, Ref} from 'react';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import RBSheet from 'react-native-raw-bottom-sheet';
+import IconClose from '../../../assets/icon/IconClose';
+import {BottomSheetV2} from '../../../components/BottomSheet';
+import dimen from '../../../utils/dimen';
+import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
+import {Button} from '../../../components/Button';
+import IconCheckCircleOutline from '../../../assets/icon/IconCheckCircleOutline';
+import useChatClientHook from '../../../utils/getstream/useChatClientHook';
+import ProfilePicture from '../../ProfileScreen/elements/ProfilePicture';
+import IconIncognito from '../../../assets/icon/IconIncognito';
+
+export type Follow = 'signed' | 'incognito' | '';
+
+interface BottomSheetFollowProps {
+  username: string;
+  profilePicture: string;
+  topicName: string;
+  isFollow: boolean;
+  setIsFollow: (isFollow: boolean) => void;
+  memberCount: number;
+  setMemberCount: (memberCount: number) => void;
+  onClose: () => void;
+  getTopicDetail: (topicName: string) => void;
+  followType: Follow;
+  setFollowType: (followType: Follow) => void;
+  ref: Ref<RBSheet>;
+}
+
+type UserItemProps = {
+  type: 'signed' | 'incognito';
+  username: string;
+  profilePicture: string;
+  isFollowing: boolean;
+  onPress?: () => void;
+};
+
+const UserItem = ({type, username, profilePicture, isFollowing, onPress}: UserItemProps) => {
```

Comment:
mas @alamsyaah can we move this to different file?
So that this file doesn't clutter much.

Reply:
bisa mas, ku pindah ya

Reply:
updated ya mas @fajarism